docs/docs/index.md

```diff
@@ -0,0 +1,73 @@
+Pathfora JS is a lightweight SDK for displaying personalized modules on your website. it integrates with your [Lytics](http://www.getlytics.com/) account to track user actions, and conditionally display modules based on your users' audience memebership. This documentation will walk you through everything you need to create your own highly customized and personalized module. Code and live preview examples are provided along the way.
+
+## Getting Started
+Before you begin creating modules with Pathfora you must have add the [Lytics Javascript Tag](https://activate.getlytics.com/#/documentation/jstag_anon) to your website. The Pathfora tag must be loaded after the Lytics tag.
+
+``` html
+<!-- Your Lytics JS Tag -->
+<script src="https://api.lytics.io/api/tag/{{YOUR LYTICS API KEY}}/lio.js"></script>
+
+<!-- Pathfora Tag -->
+<script src="http://c.lytics.io/static/pathfora.min.js"></script>
```

Comment:
`lio.js` will automatically load `pathfora.min.js` when there are campaigns running in the account. Having to manually include `pathfora.min.js` is the unusual case.